theurgy.lic

```diff
@@ -319,7 +319,7 @@ class TheurgyActions
     complete_or_interrupt_research
     cast_spell({ 'abbrev' => 'bless',
                  'mana' => 1,
-                 'prep_time' => 2,
+                 'prep_time' => 4,
```

Comment:
Should this delegate to a waggle so players can account for nuanced prep time/etc for them to cast the spell?

Reply:
Turns out there is a method to quick-cast bless in common-theurgy...